src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.targets

```diff
@@ -10,6 +10,30 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
Can we move the VB `Import` definitions to `Microsoft.NET.Sdk.VisualBasic.props` instead of the targets file?  This will make it possible to override / remove these in the project file, and will probably make the project properties page that manages these work better too.

/cc @KathleenDollard

Reply:
Should we do that separately? That way if figure out there are issues with doing it, we can revert it independent of this change.

Reply:
https://github.com/dotnet/sdk/issues/19697 to track this.